seqr/utils/search/hail_search_utils.py

```diff
@@ -86,19 +86,19 @@ def _execute_lookup(variant_id, data_type,  user, **kwargs):
     return _execute_search(body, user, path='lookup', exception_map={404: 'Variant not present in seqr'}), body
 
 
-def hail_variant_lookup(user, variant_id, **kwargs):
-    variant, _ = _execute_lookup(variant_id, Sample.DATASET_TYPE_VARIANT_CALLS, user, **kwargs)
+def hail_variant_lookup(user, variant_id, dataset_type, **kwargs):
+    variant, _ = _execute_lookup(user, variant_id, data_type=dataset_type, **kwargs)
     return variant
 
 
-def hail_sv_variant_lookup(user, variant_id, samples, sample_type=None, **kwargs):
+def hail_sv_variant_lookup(user, variant_id, dataset_type, samples, sample_type=None, **kwargs):
```

Comment:
`dataset_type` as an arg is redundant here but I have it in here because the caller https://github.com/broadinstitute/seqr/pull/4097/files#diff-20e96f31f5b1390b1b519bb5fc6cefb42e859489cbc0d4d2f084b80a2ee63342R161 is shared